src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -79,7 +79,7 @@ export const mockContactInformation = {
     addressLine2: 'string',
     addressLine3: 'string',
     addressPou: 'CORRESPONDENCE',
-    addressType: 'domestic',
+    addressType: 'DOMESTIC',
```

Comment:
this now matches reality: https://github.com/department-of-veterans-affairs/vets-website/blob/0f539058825668a08f9518f56c85dec9dd96d852/src/platform/user/profile/vet360/constants/index.js#L12-L16